editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { FaviconForm } from "./FaviconFrom";
+import { TextLinkForm } from "./TextLinkForm";
+import { ThemeAndLogoForm } from "./ThemeAndLogoForm";
+
+export const DesignPreview = styled(Box)(({ theme }) => ({
+  border: `2px solid ${theme.palette.border.input}`,
+  padding: theme.spacing(2),
+  boxShadow: "4px 4px 0px rgba(150, 150, 150, 0.5)",
+}));
+
+export const EXAMPLE_COLOUR = "#007078";
+
+type SettingsFormProps = {
+  legend: string;
+  description: React.ReactElement;
+  input: React.ReactElement;
+  formik: FormikProps<any>;
+  preview?: React.ReactElement;
+};
+
+const useTeam = () => {
+  const [team, setTeam] = useState<Team>({} as Team);
+
+  useEffect(() => {
+    const fetchTeam = async () => {
+      try {
+        const fetchedTeam = await useStore.getState().fetchCurrentTeam();
+        if (!fetchedTeam) throw Error("Unable to find team");
+        setTeam(fetchedTeam);
+      } catch (error) {
+        console.error("Error fetching team:", error);
+      }
+    };
+
+    fetchTeam();
+  }, []);
+
+  return team;
+};
+
+export const SettingsForm: React.FC<SettingsFormProps> = ({
+  formik,
+  legend,
+  description,
+  input,
+  preview,
+}) => {
+  return (
+    <EditorRow background>
+      <form onSubmit={formik.handleSubmit}>
+        <InputGroup flowSpacing>
+          <InputLegend>{legend}</InputLegend>
+          {description}
+          {input}
+        </InputGroup>
+        {preview && (
+          <Box>
+            <Typography variant="h4" my={1}>
+              Preview:
+            </Typography>
+            {preview}
+          </Box>
+        )}
+        <ErrorWrapper error={getIn(formik.errors, "primaryColour")} id="design-settings-theme-error">
+          <Box>
+            <Button type="submit" variant="contained" disabled={!formik.dirty}>
+              Save
+            </Button>
+            <Button
+              onClick={() => formik.resetForm()}
+              type="reset"
+              variant="contained"
+              disabled={!formik.dirty}
+              color="secondary"
+              sx={{ ml: 1.5 }}
+            >
+              Reset changes
+            </Button>
+          </Box>
+        </ErrorWrapper>
+      </form>
+    </EditorRow>
+  );
+};
+
+const DesignSettings: React.FC = () => {
+  const isUsingFeatureFlag = hasFeatureFlag("SHOW_TEAM_SETTINGS");
+  const team = useTeam();
+
+  return (
+    <>
+      <EditorRow>
+        <Typography variant="h2" component="h3" gutterBottom>
+          Design
+        </Typography>
+        <Typography variant="body1">
+          How your service appears to public users
+        </Typography>
+      </EditorRow>
+      {!isUsingFeatureFlag ? (
+        <EditorRow>
+          <FeaturePlaceholder title="Feature in development" />{" "}
+        </EditorRow>
+      ) : (
+        <>
+          <ThemeAndLogoForm team={team} />
+          <ButtonForm />
+          <TextLinkForm />
+          <FaviconForm />
```

Comment:
ButtonForm, FaviconForm and TextLinkForm are currently still placeholders are will be wired up in follow-up PR sbranched from this one. The current PR is big enough as is!